doc/source/multivisit_api.rst

```diff
@@ -0,0 +1,59 @@
+.. _multivisit_api:
+****************************
+API for MVM Processing Code
+****************************
+
+The ``hapmultisequencer.py`` module is part of the drizzlepac package and calls
+utilities which are intended to be used with
+Multi-visit Mosaic (MVM) data and to be imported into other Python programs or
+a Python session for interactive data analysis.
+
+.. toctree::
+   :maxdepth: 1
+
+hapmultisequencer
+---------------------
+Primary user interface for performing MVM processing.
+
+.. automodule:: drizzlepac.hapmultisequencer
+.. autofunction:: drizzlepac.hapmultisequencer.run_mvm_processing
+
+haputils.cell_utils
+-------------------
+Module for defining ProjectionCells and SkyCells for a set of exposures.
+
+.. automodule:: drizzlepac.haputils.cell_utils
+.. autofunction:: drizzlepac.haputils.cell_utils.get_sky_cells
+.. autofunction:: drizzlepac.haputils.cell_utils.interpret_scells
+.. autoclass:: drizzlepac.haputils.cell_utils.ProjectionCell
+.. autoclass:: drizzlepac.haputils.cell_utils.SkyCell
```

Comment:
While this is an API, it sort of seems like it really represents (or should represent) the public functions available for folks to use for data analysis.  Not all of the functions/methods are listed here, and some of them (constructor for ProjectionCell) I am not sure is useful in this public context.  The parameters are not described in the code.

Reply:
The docstrings for the classes in cell_utils were expanded to include details on the attributes for each class.  This provides the most basic information a user might need to understand what these classes define so that they can know where to find the information they need during reprocessing. 
Fixed in 3cd7f7e.